projects/ui/src/lib/components/po-navbar/po-navbar-actions/po-navbar-action/po-navbar-action.component.html

```diff
@@ -1,3 +1,7 @@
-<div tabindex="0" class="po-navbar-action-content po-clickable" (click)="click()">
-  <span class="po-icon {{ icon }}" [p-tooltip]="tooltip"> </span>
-</div>
+<po-icon
+  (click)="click()"
+  tabindex="0"
+  class="po-navbar-action-content po-clickable"
+  [p-icon]="icon"
+  [p-tooltip]="tooltip"
+></po-icon>
```

Comment:
```suggestion
<po-icon
  tabindex="0"
  class="po-navbar-action-content po-clickable"
  [p-icon]="icon"
  [p-tooltip]="tooltip"
  (click)="click()"
></po-icon>
```
Propriedades dos elementos devem seguir a seguinte ordem:

class (CSS)
name (nome do componente quando houver necessidade, inputs, buttons, etc…)
ngModel (quando existir)
propriedades sem bind
propriedades com bind
eventos